connector/routingconnector/README.md

```diff
@@ -47,6 +47,8 @@ The following settings are available:
 ### Supported [OTTL] functions
 
 - [IsMatch](../../pkg/ottl/ottlfuncs/README.md#IsMatch)
+- [IsString](../../pkg/ottl/ottlfuncs/README.md#IsString)
```

Comment:
Should these docs changes be more significant now that we're adding all StandardConverters?

Reply:
Ahh yep thanks for the catch/change of scope 🚀